scripts/skaffold.init.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
+GCLOUD_SDK_PATH="${HOME}/${GCLOUD_SDK_DIR:-"google-cloud-sdk"}"
+source ${GCLOUD_SDK_PATH}/path.bash.inc
+
+NS=${NAMESPACE:-${NS}}
```

Comment:
Why just don't use `${NAMESPACE}`?

Reply:
We could check the variable contains a value.

Reply:
It was the shorthand been used on desktop: `NS=smth ./skaffold.init.sh`